### PR TITLE
Fix computed property override deprecations

### PR DESCRIPTION
**Pull request by josemarluedke**
This fixes computed property override deprecations. Initial work was done in #1841 but not merged.

### CODE REVIEW

File: addon/session-stores/adaptive.js [hunk 1]
```diff
@@ -105,24 +105,20 @@ export default Base.extend({
...
-  _isLocalStorageAvailable: computed(function() {
-    try {
-      localStorage.setItem(LOCAL_STORAGE_TEST_KEY, true);
-      localStorage.removeItem(LOCAL_STORAGE_TEST_KEY);
-      return true;
-    } catch (e) {
-      return false;
-    }
-  }),
-
```
Comment (Turbo87) [inline addon/session-stores/adaptive.js]: why did you convert these computed properties? where are they overridden?

File: addon/session-stores/cookie.js [hunk 2]
```diff
@@ -135,12 +135,6 @@ export default BaseStore.extend({
 
   _cookies: service('cookies'),
 
-  _fastboot: computed(function() {
-    let owner = getOwner(this);
-
-    return owner && owner.lookup('service:fastboot');
-  }),
-
```
Comment (Turbo87) [inline addon/session-stores/cookie.js]: same as above, what is overriding this property?

File: addon/utils/is-fastboot.js [hunk 3]
```diff
@@ -10,8 +10,18 @@ import ApplicationInstance from '@ember/application/instance';
...
+      if (this.hasOwnProperty('isFastBootOverride')) {
+        return this.isFastBootOverride;
+      } else {
+        return isFastBoot(getOwner(this));
+      }
+    },
+    set(key, value) {
+      this.isFastBootOverride = value;
+      return value;
+    }
```
Comment (Turbo87) [inline addon/utils/is-fastboot.js]: I don't understand this change. Why would something call `this.set('isFastboot`, true/false)`? Seems to me that this kind of thing won't change during the runtime of the app 🤔

**Comment by josemarluedke** [comment]
@Turbo87 I have pretty much implemented the same thing as #1841. However, I do agree with the change.

Applications will probably not override these 3 cases you mentioned, but, we override these properties in tests. Looking at the console while running tests, you will see that are many deprecations. 

IMO, if Ember is deprecating something, the internal tests of a lib should not be relying on them.

Happy to revert these changes if you think we don't need them.

**Comment by Turbo87** [comment]
> we override these properties in tests

I don't think anything that is running in the browser should override the `isFastboot` flag to be `true`. Also, these flags are **private** properties and implementation details, they could change at any time so relying on them seems very brittle to me.

**Comment by josemarluedke** [comment]
@Turbo87 Sorry for my wording, but by "we" I meant ember-simple-auth. Ember Simple Auth does override these properties in tests.

**Comment by Turbo87** [comment]
oh, okay! sorry, wasn't aware of that. 😅 

I would still like to avoid making the production code more complicated just so that we can better mock things in the tests 🤔

**Comment by josemarluedke** [comment]
@Turbo87 all right, removed the other changes. Keet only the changes around the configuration.

**Comment by josemarluedke** [comment]
@Turbo87 @marcoow Ping. :D

**Pull request merged**